src/utils/validator.js

```diff
@@ -0,0 +1,5 @@
+export const checkValid = ({ email, password }) => {
+  if (!email || !email.match(/@/g)) return false;
+  if (!password || password.length < 8) return false;
+  return true;
+};
```

Comment:
유효성 검사 하는 유틸 만드신 것 좋네요! 👍

다만 앞에 부정문 ! 표시는 반대로 생각해야하는 것 때문에 가독성이 떨어져서 네이밍으로 알 수 있게 짜시고 처리하는게 좋은걸로 알고 있습니다!

```suggestion
export const checkValid = ({ email, password }) => {
  if (hasAtSign(email) && isOverMinimumLetter(password)) return true;
  return false;
};
```

Reply:
오오 확실히 조건을 파악하는 것이 수월해지네요!👍